azure-pipelines.yml

```diff
@@ -14,139 +14,48 @@ pr:
     - '*'
 
 variables:
-  - name: is-runtime-release
-    value: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/Runtime-v')]
-  - name: is-sdk-release
-    value: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/SDK-v')]
-  - name: Codeql.Enabled
-    value: true
+- name: Codeql.Enabled
+  value: true
+
+parameters:
+- name: pools
+  type: object
+  default:
+  - name: NetCore-Public
+    vmImage: ubuntu-latest # The public PR pool schema is different from the 1ES one. We use 'vmImage' which is a default azure pipeline schema.
```

Comment:
Comment on all of these seems different but not able to correlate with direct line. May be specify at the top together?

Reply:
That's a good point, these comments are kind of in the middle of nowhere. Moving them.